### ISSUE TITLE
Update the README to mention the backends

### ISSUE THREAD

**Issue by chikamichi** [nu7hatch/aclatraz#10]
Hi,

the current (master) README does not mention the different DB backend (Redis, Mongo, Cassandra…). This would be a good improvement.

**Comment by nu7hatch** [comment]
Yeah, recently i'm overworked with other stuff, but i will find some time next month to finish wiki and improve a little bit new version according to users feedback. 
Cheers!
